PI_Log.txt

```diff
@@ -0,0 +1,4 @@
+2017-04-17 23:06:25 Pi SerialOutputListener created
```

Comment:
This file shouldn't have been checked in. We should add it to the .gitignore file.